### PR TITLE
feat: move smart queue periodic task to worker

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: mergify_engine/tests/unit/test_forwards.py [hunk 1]
```diff
@@ -22,17 +22,10 @@
...
 from mergify_engine import utils
 from mergify_engine import web
 
 
-# NOTE(sileht): Celery magic, this just skip amqp and execute tasks directly
-# So all REST API calls will block and execute celery tasks directly
-tasks.app.conf.task_always_eager = True
-tasks.app.conf.task_eager_propagates = True
-
-
```
Comment (jd) [inline mergify_engine/tests/unit/test_forwards.py:35]: I've a doubt that this should be part of this PR?

Comment (sileht) [reply mergify_engine/tests/unit/test_forwards.py:35]: Good catch!

**Comment by sileht** [comment]
I have to dismiss manually is that normal ?

File: mergify_engine/tests/unit/test_forwards.py (hunk 1 above)
Comment (jd) [reply mergify_engine/tests/unit/test_forwards.py:35]: Why I still see it there?

Comment (sileht) [reply mergify_engine/tests/unit/test_forwards.py]: Because I clean base.py that was having these lines too...

**Pull request merged**